code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -498,60 +521,153 @@
 	return TRUE
 
 /datum/ritual/cruciform/divisor/divisor_smite
 	name = "Divine Smite"
 	phrase = "In maxilla asini, in mandibula pulli asinarum, delevi eos, et percussi mille viros." //"With a donkey’s jawbone, I have made donkeys of them. With a donkey’s jawbone, I have killed a thousand men"
-	desc = "A short litany spoken in the middle of battle. Considered tricky to use, as it only lasts 30 seconds, but gives the speaker additional power and strength when swinging a melee weapon. Takes five minutes to recharge."
+	desc = "A short litany spoken in the middle of battle. Gives the speaker additional power and strength when swinging a melee weapon for two minutes. Takes five minutes to recharge."
 	cooldown = TRUE
 	cooldown_time = 5 MINUTES
 	cooldown_category = "divisor_smite"
-	effect_time = 30 SECONDS
-	power = 30
+	effect_time = 90 SECONDS
+	power = 25
 	var/wrath_damage = 0.2
 
 /datum/ritual/cruciform/divisor/divisor_smite/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C)
 	user.damage_multiplier += wrath_damage
 	to_chat(user, SPAN_NOTICE("You feel divine wrath empowering you with immense but fleeting strength!"))
 	set_personal_cooldown(user)
-	addtimer(CALLBACK(src, PROC_REF(discard_effect), user), src.cooldown_time)
+	addtimer(CALLBACK(src, PROC_REF(discard_effect), user), src.effect_time)
 	return TRUE
 
 /datum/ritual/cruciform/divisor/divisor_smite/proc/discard_effect(mob/living/carbon/human/user, amount)
+	to_chat(user, SPAN_NOTICE("You feel the divine wrath fade, leaving you with only your own strength."))
 	user.damage_multiplier -= wrath_damage
 
+/datum/ritual/cruciform/divisor/zoom_litany
+	name = "Infinite Hymn"
+	phrase = "Quam pulchri super montes pedes annuntiantis et praedicantis pacem; annuntiantis bonum, praedicantis salutem, dicentis Sion: Regnabit Deus tuus!" //"How beautiful on the mountains are the feet of those who bring good news, who proclaim peace, who bring good tidings, who proclaim salvation, who say to Zion, “Your God reigns!”"
+	desc = "Empowers the speaker with enhanced movement speed, allowing them to run faster for a short time. While useful, the body must rest after exceeding its limits, normally for \
+	only a mere ten minutes."
+	cooldown = TRUE
+	cooldown_time = 5 MINUTES
+	effect_time = 1 MINUTES
+	power = 25
+	cooldown_category = "zoom_litany"
+
+/datum/ritual/cruciform/divisor/zoom_litany/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C,list/targets)
+	to_chat(user, "<span class='info'>You feel yourself speeding up, the Absolute speeding you on your way!</span>")
+	user.added_movedelay -= 0.5
+	set_personal_cooldown(user)
+	addtimer(CALLBACK(src, PROC_REF(discard_effect), user), src.effect_time)
+	return TRUE
+
+/datum/ritual/cruciform/divisor/zoom_litany/proc/discard_effect(mob/living/carbon/human/user)
+	to_chat(user, "You feel yourself slowing down once more.")
+	user.added_movedelay += 0.5
```

Comment:
```suggestion
	addtimer(CALLBACK(user, TYPE_PROC_REF(/mob/living/carbon/human, clear_movement_delay), 0.5), effect_time)
	return TRUE
```
Something like this is better

Reply:
Ehhhh... I'm not sure that's better. For one, it doesn't give the text to the user, and giving feedback in chat is good. Second, it'd need to be -0.5 anyway, since the intention is to add more delay and that proc would remove it (they'd get even faster after the callback). Third, it's just not as cleanly readable.

Reply:
Fair enuff